test/test_complex.py

```diff
@@ -10,36 +11,38 @@
 
 
 class TestComplexTensor(TestCase):
-    def test_to_list_with_complex_64(self):
+    @dtypes(*torch.testing.get_all_complex_dtypes())
+    def test_to_list(self, device, dtype):
         # test that the complex float tensor has expected values and
         # there's no garbage value in the resultant list
-        self.assertEqual(torch.zeros((2, 2), dtype=torch.complex64).tolist(), [[0j, 0j], [0j, 0j]])
+        self.assertEqual(torch.zeros((2, 2), device=device, dtype=dtype).tolist(), [[0j, 0j], [0j, 0j]])
 
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
-    def test_exp(self):
-        def exp_fn(dtype):
-            a = torch.tensor(1j, dtype=dtype) * torch.arange(18) / 3 * math.pi
-            expected = np.exp(a.numpy())
-            actual = torch.exp(a)
-            self.assertEqual(actual, torch.from_numpy(expected))
-
-        exp_fn(torch.complex64)
-        exp_fn(torch.complex128)
-
-    def test_copy_real_imag_methods(self):
-        real = torch.randn(4)
-        imag = torch.randn(4)
+    @dtypes(*torch.testing.get_all_complex_dtypes())
+    def test_exp(self, device, dtype):
```

Comment:
The tests for any op supported for complex dtypes should go in `test_torch.py` along with the other dtypes :D

Let's extend `test_exponential` in `TestTorchDeviceType` or adda new test in that class.